aws-dynamodb-table/table.tf

```diff
@@ -0,0 +1,38 @@
+resource "aws_dynamodb_table" "this" {
+  name         = var.table_name
+  billing_mode = "PAY_PER_REQUEST"
+  hash_key     = var.hash_key
+  range_key    = var.range_key != "" ? var.range_key : null
+
+  dynamic "attribute" {
+    for_each = var.attributes
+    content {
+      name = attribute.value["name"]
+      type = attribute.value["type"]
+    }
+  }
+
+  dynamic "global_secondary_index" {
+    for_each = var.global_secondary_indexes
+    content {
+      name            = global_secondary_index.value.name
```

Comment:
🤔 tady by asi mělo být each.value když iteruješ tím cyklem. global_secondary_index nevidím někde definovaný. No však si to spusť a uvidíš. 😄

Reply:
Funguje to i takhle 🥳 Pro nejakou konzistenci - chces to udelat tim tvym zpusobem?